frigate/events/maintainer.py

```diff
@@ -202,7 +202,9 @@ def handle_object_detection(
                 Event.end_time: end_time,
                 Event.zones: list(event_data["entered_zones"]),
                 Event.thumbnail: event_data["thumbnail"],
-                Event.has_clip: event_data["has_clip"],
+                Event.has_clip: event_data["has_clip"]
+                if event_data.get("max_severity")
```

Comment:
I am a bit concerned about this because it creates a situation where you might have alerts or detections enabled for 99% of the event but if you disable them when the object stops being tracked then it isn't saved.

I would rather leave this logic as is, because the has_clip value already checked the max_severity, but it is retained (once true, always true)

Reply:
makes sense, sounds good